scripts/ci/run-test-microbench.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -e
+
+export TESTCMD=`dirname "${BASH_SOURCE[0]}"`/run-step.sh
+
+${TESTCMD} --label=microbenchmark --timeout=40m make -C acceptance-tests DebianShootoutMono.stamp
```

Comment:
How long does it take to run ?

Reply:
On my machine? Each of them takes less than 3 minutes.